x-pack/plugins/maps/public/connected_components/mb_map/draw_control/draw_control.tsx

```diff
@@ -13,99 +13,130 @@ import MapboxDraw from '@mapbox/mapbox-gl-draw';
 import DrawRectangle from 'mapbox-gl-draw-rectangle-mode';
 import type { Map as MbMap } from '@kbn/mapbox-gl';
 import { Feature } from 'geojson';
-import { DRAW_TYPE } from '../../../../common/constants';
+import { DRAW_SHAPE } from '../../../../common/constants';
 import { DrawCircle } from './draw_circle';
 import { DrawTooltip } from './draw_tooltip';
 
+const SIMPLE_SELECT = 'simple_select';
```

Comment:
simple_select, draw_polygon, draw_line_string, and draw_point are all constants defined in [MapboxDraw.modes](https://github.com/mapbox/mapbox-gl-draw/blob/main/src/constants.js#L48). I don't think we should redefine them here. I would recommend removing SIMPLE_SELECT, DRAW_POLYGON, DRAW_LINE, DRAW_POINT constants and just using the values provided by `MapboxDraw.modes`.